IBPSA/Fluid/Actuators/BaseClasses/exponentialDamper_inv.mo

```diff
@@ -0,0 +1,57 @@
+within IBPSA.Fluid.Actuators.BaseClasses;
+function exponentialDamper_inv
+  "Inverse function of the exponential damper characteristics"
+  extends Modelica.Icons.Function;
+  input Real kThetaSqRt "Square root of loss coefficient";
+  input Real[:] kSupSpl "k values of support points";
+  input Real[:] ySupSpl "y values of support points";
+  input Real[:] invSplDer "Derivatives at support points";
+  output Real y "Fractional opening";
+protected
+  parameter Integer sizeSupSpl = size(kSupSpl, 1) "Number of spline support points";
+  Integer i "Integer to select data interval";
+algorithm
+  i := 1;
+  for j in 2:sizeSupSpl loop
+    if kThetaSqRt <= kSupSpl[j] then
+      i := j;
+      break;
+    end if;
+  end for;
+  y := IBPSA.Utilities.Math.Functions.smoothLimit(
+    IBPSA.Utilities.Math.Functions.cubicHermiteLinearExtrapolation(
+      x=kThetaSqRt,
+      x1=kSupSpl[i - 1],
+      x2=kSupSpl[i],
+      y1=ySupSpl[i - 1],
+      y2=ySupSpl[i],
+      y1d=invSplDer[i - 1],
+      y2d=invSplDer[i]),
+    0,
+    1,
+    1E-3);
+annotation (smoothOrder=1,
+Documentation(info="<html>
+<p>
+This function provides an approximate inverse of the exponential damper characteristics.
```

Comment:
I would add a sentence like 'This function thus computes the valve opening that corresponds to some loss coefficient.'

Reply:
Addressed